### ISSUE TITLE
Dumping Dgraph data sometimes results in incorrect RDF type

### ISSUE THREAD

**Issue by sungwoncho** [dgraph-io/dgraph#1052]
```
Dgraph version   : v0.7.7
Commit SHA-1     : 052e077
Commit timestamp : 2017-06-13 16:59:35 +1000
Branch           : HEAD
```
---

I have a schema such that:

```
...
Timestamp: datetime .
...
```

After dumping the data, when I load it using dgraphloader, I get:

```
Processing dgraph-1-2017-06-14-09-54.rdf.gz
2017/06/13 23:57:22 main.go:135: Error while parsing RDF: Unrecognized rdf type xs:datetime, on line:319740 <0x3b82dae121c9aa12> <Timestamp> "2017-06-12T23:50:53.461Z"^^<xs:datetime> .
```

**Comment by pawanrawal** [comment]
A fix has been merged to `release/v0.7.7` and `master`.

**Comment by pawanrawal** [comment]
There was a bug because of which type was dumped as `<xs:datetime>` instead of `<xs:dateTime>`. It should be fixed now.
